src/plugins/dashboard/public/dashboard_container/embeddable/dashboard_container.tsx

```diff
@@ -852,18 +818,14 @@ export class DashboardContainer
     this.dispatch.setStateFromSettingsFlyout(settings);
   };
 
-  public setExpandedPanelId = (newId?: string) => {
-    this.dispatch.setExpandedPanelId(newId);
-  };
-
   public setViewMode = (viewMode: ViewMode) => {
+    // block the Dashboard from entering edit mode if this Dashboard is managed.
+    if (this.managed$.value && viewMode?.toLowerCase() === ViewMode.EDIT?.toLowerCase()) {
```

Comment:
`ViewMode.EDIT` is already lowercase?

```suggestion
    if (this.managed$.value && viewMode?.toLowerCase() === ViewMode.EDIT) {
```